internal/issue_trackers/github/github.go

```diff
@@ -74,26 +72,16 @@ func (g *Github) GetIssue(identifier string) (issue domain.Issue, err error) {
 		}
 	}
 
-	return domain.Issue{
-		ID:           strconv.FormatInt(result.Number, 10),
-		Title:        result.Title,
-		Body:         result.Body,
-		Url:          result.Url,
-		Labels:       labels,
-		IssueTracker: domain.IssueTrackerTypeGithub,
+	return Issue{
+		id:           strconv.FormatInt(result.Number, 10),
+		title:        result.Title,
+		body:         result.Body,
+		url:          result.Url,
+		labels:       labels,
+		labelsConfig: g.cfg.IssueLabels,
```

Comment:
thought: config embed in this model looks really strange.

Reply:
Inject single label on issue creation